src/test/java/com/epam/page/object/generator/validators/AbstractValidatorTest.java

```diff
@@ -0,0 +1,70 @@
+package com.epam.page.object.generator.validators;
+
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.epam.page.object.generator.model.SearchRule;
+import com.epam.page.object.generator.utils.SearchRuleType;
+import com.google.common.collect.Sets;
+import org.assertj.core.util.Lists;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+public class AbstractValidatorTest {
```

Comment:
I think it's worth adding more tests with multiple search rules, because we iterating over a list of search rules in SUT

Reply:
Done